### DISCUSSION TITLE
Overlapping Componant found, please fix it

### DISCUSSION THREAD

**Discussion by Shariar-Hasan** [community/community#55199] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

The welcome message is not showing properly.
all this components overlapping the workflow list.
for example:
![image](https://github.com/community/community/assets/75330767/6f4c92b5-4628-470b-9912-eeb6d4dbe220)

fix it. I can't even close the welcome message.

**Comment by MansiAyer** [comment]
same issue on my feed too

**Reply by Shariar-Hasan** [reply]
i hope they will fix it soon

**Reply by jareehmonda** [reply]
> ### Select Topic Area
> Bug
> 
> ### Body
> The welcome message is not showing properly. all this components overlapping the workflow list. for example: ![image](https://user-images.githubusercontent.com/75330767/237747370-6f4c92b5-4628-470b-9912-eeb6d4dbe220.png)
> 
> fix it. I can't even close the welcome message.
Thanks for posting this in the community. Because the same problem was happening to me 4 weeks ago. But I just saw [this post](https://thavatcalculator.com/) and I was very happy to see it. Because now my problem is also solved.

**Comment by joyously** [comment]
I came here to report a similar case.
The sidebar is overlapping the main content of the profile landing page.
![GitHub-45](https://github.com/community/community/assets/7957680/ff77ad3c-e27c-4378-9437-a2a621b8952f)

**Reply by Shariar-Hasan** [reply]
i hope they will fix it soon

**Comment by thispsj** [comment]
Yes this really needs a fix. It becomes frustrating to open the dashboard at times. So most of the times, I'm like directly entering my personal profile URL and going to repos from there.

**Reply by Shariar-Hasan** [reply]
yeah, same here, kind of annoying to me 😔

**Comment by Wahid7852** [comment]
Same on the mobile version
the feed is bugged on both platforms

**Reply by Shariar-Hasan** [reply]
this should be fixed

**Comment by t-var-s** [comment]
Having the same issue
<img width="378" alt="image" src="https://github.com/community/community/assets/6660327/94235889-3c1b-4c60-9567-b4789ca39e94">

**Reply by Shariar-Hasan** [reply]
its kind of annoying 😔

**Comment by sh-lucas** [comment]
I have the same problem.
Firefox 113.0.1 (64 bits), started tonight.
1366x768 screen

**Reply by Shariar-Hasan** [reply]
i think, its kind of issue in almost all browser.
isnt it?

**Comment by mmiscool** [comment]
This is extremely annoying.

**Reply by Shariar-Hasan** [reply]
yeah, i hope they will fix it soon

**Comment by Praiz001** [comment]
I came here to report similar issue too. Feed and sidebar are overlapping.

**Reply by Shariar-Hasan** [reply]
i hope you get it fixed. maybe they have fixed it

**Comment by thispsj** [comment]
I think @github has fixed this for Desktop. Mobile site still overlaps the feed.

**Reply by joyously** [reply]
> I think @github has fixed this for Desktop. Mobile site still overlaps the feed.

I just checked again using Firefox 113 on Ubuntu 22.04, refreshing twice, and it still overlaps.

**Reply by thispsj** [reply]
Hi @joyously, maybe you are right but on my Chrome 113 on Windows 11 it was overlapping some days back but when I opened GitHub today, it was fixed. So maybe @github is working on a fix and it will be available to all very soon.

**Reply by Shariar-Hasan** [reply]
my one also got fixed, in Microsoft edge.

**Reply by cellio** [reply]
Chrome on MacOS (desktop), not fixed as of now.

**Reply by Shariar-Hasan** [reply]
maybe they have fixed it, you can check now

**Reply by Shariar-Hasan** [reply]
> Chrome on MacOS (desktop), not fixed as of now.

its showing the same bug again :)
![image](https://github.com/community/community/assets/75330767/59d515ff-fc63-4266-ad36-c83a04815434)

**Comment by cellio** [comment]
I came here to report this too -- Chrome, consistent over several days:

![Screen Shot 2023-05-18 at 5 43 44 PM](https://github.com/community/community/assets/5557942/08b44924-6221-478d-a074-500875e9d687)

**Reply by Shariar-Hasan** [reply]
maybe they have fixed it, can you check now?

**Reply by cellio** [reply]
No change for me yet, alas.

**Reply by Shariar-Hasan** [reply]
thats very sad, i hope you get the fixs

**Comment by saubhagya12k4** [comment]
github plz fix this bug

**Reply by Shariar-Hasan** [reply]
i guess they are fixing, as in my microsoft edge its working
![image](https://github.com/community/community/assets/75330767/325fa985-7d52-4ee2-b073-d7ac4a1fd80e)

**Comment by jayantur13** [comment]
Similar issue in Ms. Edge

**Reply by Shariar-Hasan** [reply]
i guess they are fixing, as in my microsoft edge its working
![image](https://github.com/community/community/assets/75330767/325fa985-7d52-4ee2-b073-d7ac4a1fd80e)

**Reply by jayantur13** [reply]
Its looking good for me now. Lets see for how long.

**Reply by Shariar-Hasan** [reply]
Really!🙂

**Comment by Shariar-Hasan** [comment]
wow github nice,
it was fixed some days ago, and now its again showing the same bug 😑😑😑
![image](https://github.com/community/community/assets/75330767/4c819959-a3ce-422d-9aed-18c5654f5d60)

**Reply by thispsj** [reply]
Yes @Shariar-Hasan 
Its back again.

**Reply by Shariar-Hasan** [reply]
exactly,  mine too😢

**Comment by aelmsly22** [comment]
I'm having the same issue, just here to mention it again!

**Reply by Shariar-Hasan** [reply]
dont know what are they doing! I got a fix some days ago. But today the bug returned again🙂

**Comment by kom0055** [comment]
It looks so uncomfortable

**Reply by Shariar-Hasan** [reply]
exactly 😥 dont know why they aren't fixing it

**Comment by mwprogrammer** [comment]
I noticed that this issue is mostly on the For you page only and that it manifests in when the cards are of a certain width. Had this issue and today I have smaller cards in my feed and it seems to be fixed. Hope it's fixed better now.

**Reply by Shariar-Hasan** [reply]
Well my issue fixed  almost a week ago, but after some days, i faced the same issue again, and still its there🙂
